### PR TITLE
Post status links for repeatable posts

### PR DESCRIPTION
**Pull request by dashaluna**
Related to #11 

Adds two table view links for `Repeating` and `Repeat` posts:

screenshot 2017-12-20 15 47 54

### CODE REVIEW

File: hm-post-repeat.php [hunk 1]
```diff
@@ -440,3 +449,142 @@ function get_repeating_post( $post_id ) {
...
+			'post_type'      => get_current_screen()->post_type,
+			'hm-post-repeat' => $repeat_type,
+		);
+
+		// Custom WP_Query to get posts count of repeat type.
+		$repeat_type_query = new \WP_Query( get_repeat_type_query_params( $repeat_type ) );
+
+		$link_label = sprintf(
+				'%s <span class="count">(%s)</span>',
+			$repeat_desc,
```
Comment (willmot) [inline hm-post-repeat.php]: I _think_ this needs escaping as is being output in html, or are these links escaped later by core?

File: hm-post-repeat.php [hunk 2]
```diff
@@ -440,3 +449,142 @@ function get_repeating_post( $post_id ) {
...
+	// Construct custom query to get posts of specified repeat type.
+	$query['meta_query'] = array(
+			array(
+					'key'     => 'hm-post-repeat',
+					'compare' => 'EXISTS',
+			),
+	);
+
+	if ( $repeat_type === 'repeat' ) {
+		$query['post_parent__not_in'] = array( 0 );
```
Comment (willmot) [inline hm-post-repeat.php]: smart!

File: hm-post-repeat.php (hunk 1 above)
Comment (dashaluna) [reply hm-post-repeat.php]: Hm, doesn't look like it - I'll add.

**Comment by dashaluna** [comment]
Review & merge :)

**Comment by dashaluna** [comment]
🚀

**Comment by willmot** [comment]
Here comes the merge train 🚅 🚅 🚅

**Comment by willmot** [comment]
We should tag a release soon too

**Comment by dashaluna** [comment]
💪 thanks!
Let's wait for specific Repeat posts display ;)

**Pull request merged**